explainability-service/src/main/resources/application.properties

```diff
@@ -49,12 +49,24 @@ quarkus.banner.path=banner.txt
 quarkus.http.handle-100-continue-automatically=true
 
 # Feature flags
+# default/dev endpoints
+endpoints.fairness=enable
```

Comment:
The dev and prod profiles have been split into `application-dev.properties` and `application.properties`, respectively. Perhaps this could be added to the `application-dev.properties` too?